src/Sarif.Driver/Sarif.Driver.csproj

```diff
@@ -11,6 +11,22 @@
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.props))\build.props" />
 
+  <ItemGroup>
+    <EmbeddedResource Update="DriverResources.resx">
```

Comment:
>DriverResources [](start = 30, length = 15)

Still yet another case where, when we converted our project files to .NET Core format, we lost the ability to edit the .resx file and have the .Designer.cs file automatically updated. I thought we had them all...